src/flatpak.types.ts

```diff
@@ -39,7 +39,7 @@ export type BuildSystem = 'meson' | 'cmake' | 'cmake-ninja' |
 export interface Module {
     name: string
     buildsystem?: BuildSystem
-    'config-opts': string[]
+    'config-opts'?: string[]
```

Comment:
It was triggering the lint [here](https://github.com/bilelmoussaoui/flatpak-vscode/blob/fced53b292abbcba9823143357d7c7b842624dcb/src/manifest.ts#L284). I'm not sure if this is undefine-able or that line needs refactoring.